code/modules/uplink/uplink_items/dangerous.dm

```diff
@@ -0,0 +1,101 @@
+	name = "Conspicuous Weapons"
+	weight = 9
+
+/datum/uplink_item/dangerous
+	category = /datum/uplink_category/dangerous
+
+// No progression cost
+
+/datum/uplink_item/dangerous/foampistol
+	name = "Toy Pistol with Riot Darts"
+	desc = "An innocent-looking toy pistol designed to fire foam darts. Comes loaded with riot-grade \
+			darts effective at incapacitating a target."
+	item = /obj/item/gun/ballistic/automatic/pistol/toy/riot
+	cost = 2
+	surplus = 10
+
+// Low progression cost
+
+/datum/uplink_item/dangerous/throwingweapons
+	name = "Box of Throwing Weapons"
+	desc = "A box of shurikens and reinforced bolas from ancient Earth martial arts. They are highly effective \
+			throwing weapons. The bolas can knock a target down and the shurikens will embed into limbs."
+	progression_minimum = 10 MINUTES
+	item = /obj/item/storage/box/syndie_kit/throwing_weapons
+	cost = 3
+	illegal_tech = FALSE
+
+/datum/uplink_item/dangerous/sword
+	name = "Energy Sword"
+	desc = "The energy sword is an edged weapon with a blade of pure energy. The sword is small enough to be \
+			pocketed when inactive. Activating it produces a loud, distinctive noise."
+	progression_minimum = 20 MINUTES
+	item = /obj/item/melee/energy/sword/saber
+	cost = 8
+	purchasable_from = ~UPLINK_CLOWN_OPS
+
+/datum/uplink_item/dangerous/powerfist
+	name = "Power Fist"
+	desc = "The power-fist is a metal gauntlet with a built-in piston-ram powered by an external gas supply.\
+			Upon hitting a target, the piston-ram will extend forward to make contact for some serious damage. \
+			Using a wrench on the piston valve will allow you to tweak the amount of gas used per punch to \
+			deal extra damage and hit targets further. Use a screwdriver to take out any attached tanks."
+	progression_minimum = 20 MINUTES
+	item = /obj/item/melee/powerfist
+	cost = 6
+
+/datum/uplink_item/dangerous/rapid
+	name = "Gloves of the North Star"
+	desc = "These gloves let the user punch people very fast. Does not improve weapon attack speed or the meaty fists of a hulk."
+	progression_minimum = 20 MINUTES
+	item = /obj/item/clothing/gloves/rapid
+	cost = 8
+
+
+// Medium progression cost
+
+/datum/uplink_item/dangerous/doublesword
+	name = "Double-Bladed Energy Sword"
+	desc = "The double-bladed energy sword does slightly more damage than a standard energy sword and will deflect \
+			all energy projectiles, but requires two hands to wield."
+	progression_minimum = 30 MINUTES
+	item = /obj/item/dualsaber
+
+	cost = 16
+	purchasable_from = ~UPLINK_CLOWN_OPS
+
+/datum/uplink_item/dangerous/doublesword/get_discount()
+	return pick(4;0.8,2;0.65,1;0.5)
+
+/datum/uplink_item/dangerous/guardian
+	name = "Holoparasites"
+	desc = "Though capable of near sorcerous feats via use of hardlight holograms and nanomachines, they require an \
+			organic host as a home base and source of fuel. Holoparasites come in various types and share damage with their host."
+	progression_minimum = 30 MINUTES
```

Comment:
I'll be honest, by the 30 minute mark, you probably _do_ have ghosts around for holoparasites but this is a really late point for them to be coming out. Many holoparasites are about large scale planning around their powers, so you'd need to be really certain you can get everything done in the effective 10 minutes left (if we assume the average round should be 40 minutes).

I get Standard worsens the perception of Holoparasites somewhat, but they aren't just weapons. They're really more enablers for shenanigans.

Reply:
for the current average life expectancy of a round sure, 40 minutes is normal, but IIRC the framework was designed to cater into the fabled 90 minute rounds